src/routes/safe/components/Transactions/helpers/TxModalWrapper/index.tsx

```diff
@@ -181,14 +185,24 @@ export const TxModalWrapper = ({
           <Container>
             {showCheckbox && <ExecuteCheckbox onChange={setExecutionApproved} />}
 
+            {!isSpendingLimitTx && canTxExecute && (
```

Comment:
`canTxExecute` should probably be `doExecute`, so that this block disappears when the checkbox is unchecked?

Reply:
I've changed the condition to use `doExecute`